node/builder.go

```diff
@@ -363,6 +364,7 @@ var ChainNode = Options(
 var MinerNode = Options(
 	// API dependencies
 	Override(new(api.Common), From(new(common.CommonAPI))),
+	Override(new(api.Net), From(new(net.NetAPI))),
```

Comment:
In no-market mode this can be provided with `Override(new(api.Net), api.NetStub{})` until we do more of the api-splitting work